tests/protocol/test_hostplugin.py

```diff
@@ -163,7 +163,7 @@ def create_mock_protocol():
             yield protocol
 
     @patch("azurelinuxagent.common.protocol.healthservice.HealthService.report_host_plugin_versions")
-    @patch("azurelinuxagent.ga.update.restutil.http_get")
```

Comment:
This is sad. We were mocking restutil.http_get via the update module, which is not even used by this test.

Reply:
yeah, I have seen these few places, thanks for correcting.